lib/cli/Cargo.toml

```diff
@@ -90,11 +110,13 @@ sha2 = "0.10.6"
 object = "0.30.0"
 wasm-coredump-builder = { version = "0.1.11", optional = true }
 tracing = { version = "0.1" }
-tracing-subscriber = { version = "0.3", features = [ "env-filter", "fmt" ] }
+tracing-subscriber = { version = "0.3", features = ["env-filter", "fmt"] }
 async-trait = "0.1.68"
 tokio = { version = "1.28.1", features = ["macros", "rt-multi-thread"] }
 once_cell = "1.17.1"
 indicatif = "0.17.5"
+opener = "0.6.1"
+hyper = { version = "0.14.27", features = ["full"] }
```

Comment:
Do we really need all the features of hyper? (I think we might be able to use only the strictly required ones, might slim down also build times)

Reply:
Yes, I just did that for a good starting measure but forgot to remove those. I'll remove them now